index.js

```diff
@@ -70,7 +95,41 @@ class Main {
     metrics.currentBlock.set(this.worker.lastConfirmedBlock);
     metrics.requestsCounter.inc(this.worker.getNewRequestsCount());
     metrics.requestsResponseTime.observe(new Date() - this.worker.lastRequestStartTime);
-    metrics.lastExportedBlock.set(this.worker.lastExportedBlock);
+    if (BLOCKCHAIN === 'eth') {
```

Comment:
Not sure why this logic can't be unified. The 'eth' case can still have `this.worker.lastExportedBlock` set to the actual block that is written in Kafka.